cg/constants/constants.py

```diff
@@ -38,8 +38,10 @@ class CaseActions(StrEnum):
     HOLD: str = "hold"
     RUNNING: str = "running"
 
+    @classmethod
+    def actions(cls) -> list[str]:
+        return [action for action in cls]
```

Comment:
Might be possible to just do `list(CaseActions)` instead. Seemed to work in a scratch file I created.